### PR TITLE
[Docs][Kuberay] Update Ray version and sample config files for Grafana

### PR DESCRIPTION
**Pull request by MortalHappiness**
## Why are these changes needed?

The documentation in KubeRay integration with Grafana is out of date.

Changes:
- Update Ray version from 2.5.0 to 2.9.0
- Update the config files from single config file to multiple config files under the `config/grafana` directory in the kuberay repo.

## Related issue number

N/A

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [x] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [x] This PR is not tested :(

### CODE REVIEW

File: doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md [hunk 1]
```diff
@@ -340,7 +340,7 @@ Refer to [this Grafana document](https://grafana.com/tutorials/run-grafana-behin
   * Click "Import".
   * Click "Upload JSON file".
   * Choose a JSON file.
-    * Case 1: If you are using Ray 2.5.0, you can use [config/grafana/default_grafana_dashboard.json](https://github.com/ray-project/kuberay/blob/master/config/grafana/default_grafana_dashboard.json).
+    * Case 1: If you are using Ray 2.9.0, you can use [the sample config files on GitHub repo](https://github.com/ray-project/kuberay/tree/master/config/grafana) (those files that are named `xxx_grafana_dashboard.json`.
```
Comment (aslonnie) [inline doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md:343]: @angelinalg can say more, but fwiw:

- use full form: "GitHub repo" -> "GitHub repository"
- "on GitHub repo" -> "on GitHub" or "in GitHub repository"
- "(those files that are named ..." -> "These file name pattern is ...", or "The file names have a pattern of ..."
  - avoid using past tense
  - remove the parenthesis, or at least match it properly.

File: doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md [hunk 2]
```diff
@@ -340,7 +340,7 @@ Refer to [this Grafana document](https://grafana.com/tutorials/run-grafana-behin
...
     * Case 2: Otherwise, you should import the `default_grafana_dashboard.json` file from `/tmp/ray/session_latest/metrics/grafana/dashboards/` in the head Pod. You can use `kubectl cp` to copy the file from the head Pod to your local machine.
```
Comment (kevin85421) [inline doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md:344]: You also need to update "Case 2". Currently, the head Pod not only generates `default_grafana_dashboard.json` but also JSON files for Ray Data and Ray Serve Grafana dashboards.

Comment (kevin85421) [reply doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md:344]: I will ping our Ray doc team later after you address this.

File: doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md (hunk 1 above)
Comment (MortalHappiness) [reply doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md]: Done

File: doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md (hunk 2 above)
Comment (MortalHappiness) [reply doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md]: Done

File: doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md [hunk 3]
```diff
@@ -340,8 +340,8 @@ Refer to [this Grafana document](https://grafana.com/tutorials/run-grafana-behin
   * Click "Import".
   * Click "Upload JSON file".
   * Choose a JSON file.
-    * Case 1: If you are using Ray 2.5.0, you can use [config/grafana/default_grafana_dashboard.json](https://github.com/ray-project/kuberay/blob/master/config/grafana/default_grafana_dashboard.json).
-    * Case 2: Otherwise, you should import the `default_grafana_dashboard.json` file from `/tmp/ray/session_latest/metrics/grafana/dashboards/` in the head Pod. You can use `kubectl cp` to copy the file from the head Pod to your local machine.
+    * Case 1: If you are using Ray 2.9.0, you can use [the sample config files in GitHub repository](https://github.com/ray-project/kuberay/tree/master/config/grafana) The file names have a pattern of `xxx_grafana_dashboard.json`.
```
Comment (kevin85421) [inline doc/source/cluster/kubernetes/k8s-ecosystem/prometheus-grafana.md:343]:
```suggestion
    * Case 1: If you are using Ray 2.9.0, you can use [the sample config files in GitHub repository](https://github.com/ray-project/kuberay/tree/master/config/grafana). The file names have a pattern of `xxx_grafana_dashboard.json`.
```

**Comment by kevin85421** [comment]
I have already requested our doc team to review this PR.

**Review by angelinalg — APPROVED** [review]
Thanks for maintaining the Ray docs!

**Review by kevin85421 — APPROVED** [review]

**Pull request merged**